valve-monitor/web-app/src/pages/index.tsx

```diff
@@ -124,6 +135,17 @@ const Home: NextPage<HomeData> = ({ valveMonitorConfig, err }) => {
                         <h3 className={styles.sectionTitle}>
                           Individual Controls
                         </h3>
+                        {valveMonitorDevices.filter(
+                          (device) => device.deviceAlarm
+                        ).length > 0 && (
+                          <Button
+                            type="primary"
+                            danger
+                            onClick={clearAllAlarms}
+                          >
+                            Clear Alarms
```

Comment:
disable this button after first click, I accidentally clicked it twice and while it cleared the alarms it also popped the error message at the top of the page

Reply:
Good call. I added `disabled={isLoading}` which isn’t perfect but seems to work good enough for this.